packages/visualizations/src/components/MapPoi/constants.ts

```diff
@@ -0,0 +1,29 @@
+import { ColorScaleTypes } from '../types';
+import type { ColorScale, Color } from '../types';
+import type { CircleLayer } from './types';
+
+export const DEFAULT_COLORS: Record<string, Color> = {
+    Default: '#CCCCCC',
+    ShapeOutline: '#FFFFFF',
+    LightGrey: '#CBD2DB',
+    DarkGrey: '#515457',
+    Blue: '#142E7B',
+} as const;
+
+export const DEFAULT_COLORSCALE: ColorScale = {
```

Comment:
This doesn't seem like it is used anywhere. I don't think we have use cases planned that may require a color scale as well (we don't do choropleth-like colors, and coloring by categories are not scales but value -> color mapping). Was it a remnant from something else?

Reply:
You're right, I've forgot to remove it ! Thanks !